index.bs

```diff
@@ -66,6 +66,88 @@ The primary goal is to make as much of this process as possible consistent betwe
 
 <h2 id="conventions">Conventions</h2>
 
+<h3 id="well-known-intrinsic-objects">Well-Known Intrinsic Objects</h3>
+
+Well-known intrinsics are built-in objects that are explicitly referenced by the algorithms of this specification and which usually have Realm specific identities. Unless otherwise specified each intrinsic object actually corresponds to a set of similar objects, one per Realm.
+
+Within this specification a reference such as %name% means the intrinsic object, associated with the current Realm, corresponding to the name. Determination of the current Realm and its intrinsics is described in ES2015, 8.3. The well-known intrinsics are listed in Table X.
+
+<table>
+  <thead>
+    <tr>
+      <th>Intrinsic Name</th>
+      <th>Global Name</th>
+      <th>Description (<em>non-normative</em>)</th>
+    </tr>
+  </thead>
+  <tr>
```

Comment:
I don't believe we need a %Loader% or %LoaderPrototype% intrinsic. `Reflect.Loader` and `Reflect.Loader.prototype` should not have any particular powers or connection to the realm, and nothing in the semantics needs to refer to them; they're just regular library functions like, say, `parseInt`.